core/gas.h

```diff
@@ -22,6 +22,14 @@ struct gasmix {
 static const struct gasmix gasmix_invalid = { { -1 }, { -1 } };
 static const struct gasmix gasmix_air = { { 0 }, { 0 } };
 
+enum gastype {
+	GASTYPE_AIR,
+	GASTYPE_NITROX,
+	GASTYPE_NORMOXIC,
+	GASTYPE_TRIMIX,
+	GASTYPE_COUNT
```

Comment:
I realize that this is missing another significant gas. **Oxygen** (O2 >= 98%)

Reply:
I added oxygen, but would suggest that you come to a conclusion with respect to hypoxic with @willemferguson.